Modules/ThirdParty/HDF5/src/itkhdf5/CMakeLists.txt

```diff
@@ -1,14 +1,18 @@
-cmake_minimum_required (VERSION 3.10)
+cmake_minimum_required (VERSION 3.12)
 project (HDF5 C)
 # ITK --start
 if(POLICY CMP0083)
```

Comment:
This ITK-specific addition can now be removed, given that the identical thing is added just below in the upstream code.

Reply:
Good catch. It didn't generate a conflict, so I never looked that closely.  Should I add a new commit to remove that? Otherwise I'm not sure which it should be squashed into...

Reply:
A new commit is fine, preferably at the end.

Reply:
Perfect, that's the easiest too. :)  Done.